django_declarative_apis/machinery/filtering.py

```diff
@@ -109,20 +125,55 @@ def _get_filtered_field_value(
 
 # TODO: make this method less complex and remove the `noqa`
 def _apply_filters_to_object(  # noqa: C901
-    inst, filter_def, expand_children=None, klass=None
+    inst,
+    filter_def,
+    expand_children=None,
+    klass=None,
+    filter_cache=None,
+    debug_indent="",
```

Comment:
I think this is `separator` rather than `indent`. Have I misunderstood the intent?

Reply:
it's an indent, it just helps to make the logs easier to match up with what level of the response object the filters are working on.  We can probably take the debug logging out now that the filters are working.